src/solvers/dgsem_p4est/dg_2d_parallel.jl

```diff
@@ -0,0 +1,208 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+
+
+
+function rhs!(du, u, t,
```

Comment:
How is this function different from the one for `ParallelP4estMesh`?

Reply:
Do you mean `ParallelTreeMesh`? If that's the case, the difference is that the `ParallelP4estMesh` solver does not support MPI mortars yet, hence the `rhs!` functions are slightly different. I could add dummy functions for the `ParallelP4estMesh` though, that should make it possible to re-use the `ParallelTreeMesh` `rhs!`. Would you prefer that?

Reply:
Nope, you're right. We can remove this function once we have mortar support in the `ParallelP4estMesh`. Could you please add a TODO comment that states this and then resolve this convo?

Reply:
Done in 1981be8.